ci-operator/step-registry/cucushift/upgrade/healthcheck/cucushift-upgrade-healthcheck-commands.sh

```diff
@@ -72,7 +67,7 @@ function check_clusteroperators() {
         echo >&2 "$unavailable_operator"
         (( tmp_ret += 1 ))
     fi
-    if ${OC} get clusteroperator -o json | jq '.items[].status.conditions[] | select(.type == "Available") | .status' | grep -iv "True"; then
+    if ${OC} get clusteroperator -o jsonpath='{.items[].status.conditions[?(@.type=="Available")].status}'| grep -iv "True"; then
```

Comment:
I recall some old versions don't support jsonpath. Would it be used in chain upgrade?

Reply:
Actually I do not worried about jsonpath compatibility in this step, because the step generally is executed in the last hop upgrade. The version on the last hop should be new enough, jsonpath should be already supported. So for your question - "Would it be used in chain upgrade", yeah, it is being used, but only used in the last hop upgrade, so I think we do not hit such problem.

Instead of that, I had the same concern as yours in chain upgrade (`cucushift-chainupgrade-toimage`). Yesterday I validated the similar case in another PR when enhancing chain upgrade process, here is a pr-rehearse job [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/50219/rehearse-50219-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-4.15-upgrade-from-stable-4.1-aws-ipi-f30/1772573332542066688/artifacts/aws-ipi-f30/cucushift-chainupgrade-toimage/build-log.txt) upgrade from 4.1, to be a little surprised, it worked perfectly.  

```
Client Version: openshift-clients-4.2.36-202006211650.p0
```